DEX/test/loop.js

```diff
@@ -2,8 +2,9 @@ const { ApiPromise, WsProvider } = require('@polkadot/api');
 
 const sleep = async time => new Promise((resolve) => setTimeout(resolve, time));
 
-const loop = async (interval = 2000) => {
-  const provider = new WsProvider('ws://127.0.0.1:9944');
+const loop = async (interval = 1000) => {
+  const ENDPOINT_URL = process.env.ENDPOINT_URL || 'ws://127.0.0.1:9944';
+  const provider = new WsProvider(ENDPOINT_URL);
```

Comment:
This needs to be reflected in the READMEs, so I'll add the commit that does this.